revscores/languages/portuguese.py

```diff
@@ -1064,21 +1064,20 @@
     "zueira"
 ])
 
-class Portuguese(Language):
+def is_badword(word):
+    return STEMMER.stem(word).lower() in BADWORDS
+                
+def is_misspelled(word):
+    if word in ("a", "A"): return False
```

Comment:
Gotcha.  I'll add "o" for the time being.  We should probably open new tickets for new bugs :)